packages/beacon-node/src/monitoring/clientStats.ts

```diff
@@ -187,7 +187,7 @@ function createValidatorStats(): ClientStats {
     }),
     validatorActive: new MetricProperty({
       jsonKey: "validator_active",
-      metricName: "vc_indices_count",
+      metricName: "vc_active_validators_count",
```

Comment:
to get value from a metric based on label see `libp2p_data_transfer_bytes_total` as an example

https://github.com/ChainSafe/lodestar/blob/9c7c08b4adf9d04cd2e5f1084ed999a1a230a422/packages/beacon-node/src/monitoring/clientStats.ts#L112-L119